managed/services/telemetry/config.default.yml

```diff
@@ -566,3 +566,43 @@ telemetry:
     data:
       - metric_name: "k8s_clusters_count"
         column: "count"
+
+  # Advisor
+  - id: AdvisorChecksDownloadedCount
+    source: VM
+    query: pmm_managed_checks_checks_downloaded_total
```

Comment:
pmm_managed_checks_checks_downloaded_total -> pmm_server_advisors_downloaded_total ?

and others the same way